app/src/main/java/com/example/hangouts/homeScreen/UserUI.java

```diff
@@ -0,0 +1,25 @@
+package com.example.hangouts.homeScreen;
+
+import com.example.hangouts.models.User;
+import com.parse.ParseUser;
+
+public class UserUI {
+
+    ParseUser parseUser;
+
+    public UserUI (ParseUser parseUser){
+        this.parseUser = parseUser;
+    }
+
+    public String getName(){
+        return parseUser.getString(User.KEY_NAME);
+    }
+
+    public String getLastName(){
+        return parseUser.getString(User.KEY_LASTNAME);
+    }
+
+    public String getLastInitial(){
+        return getLastName().substring(0, 1);
```

Comment:
Let's add checking for an empty string here.  There may be validation on user creation or backend to prevent, but its a good practice to code defensively to avoid crashes. 
if (getLastName().length == 0) {
return "";
} else {
        return getLastName().substring(0, 1);
        }